webknossos-datastore/app/com/scalableminds/webknossos/datastore/controllers/BinaryDataController.scala

```diff
@@ -251,22 +251,22 @@ class BinaryDataController @Inject()(
   }
 
   /**
-    * Handles isosurface requests.
+    * Handles ad-hoc mesh requests.
     */
   @ApiOperation(hidden = true, value = "")
-  def requestIsosurface(token: Option[String],
-                        organizationName: String,
-                        dataSetName: String,
-                        dataLayerName: String): Action[WebKnossosIsosurfaceRequest] =
-    Action.async(validateJson[WebKnossosIsosurfaceRequest]) { implicit request =>
+  def requestAdHocMesh(token: Option[String],
+                       organizationName: String,
+                       dataSetName: String,
+                       dataLayerName: String): Action[WebKnossosAdHocMeshRequest] =
```

Comment:
```suggestion
                       dataLayerName: String): Action[WebknossosAdHocMeshRequest] =
```

While whe’re here :sweat_smile: The camel case spelling is deprecated